h/services/group.py

```diff
@@ -62,6 +62,9 @@ def create(self, name, authority, userid, description=None, type_='private'):
         for attr, value in access_flags.iteritems():
             setattr(group, attr, value)
 
+        if group.joinable_by is not None:
+            group.members.append(creator)
```

Comment:
I think this is better than `if type_ != 'publisher'` because it's using the underlying logic and not the abstraction level that is the group types.